docs/troposphere_iz.rst

```diff
@@ -0,0 +1,20 @@
+troposphere modules i - z
```

Comment:
This seems to be missing quite a few modules (rds, s3 to name a couple).

Also, should these files be auto-generated?

Reply:
BTW, also noticed it was missing "h" between "a-g" and "i-z". And, as an example of autogenerating the list, something like this is quick and dirty:
```
ls ../troposphere/[a-i]*.py | sed 's#../troposphere/\(.*\).py#\1#' | xargs printf ".. automodule:: troposphere.%s\n    :members:\n\n"
```

Reply:
Agree with the these should be autogenerated but I wasn't sold on my original organization of the files alphabetically.  After some thought, it might be better to organize the modules as AWS does, i.e. by Compute, Storage, Database, Networking, etc.  While more work it would be more native and possibly flow better for first time users.

An autogenerated alphabetical file could be added to ensure complete module coverage and provide an additional way to browse the libraries.

Reply:
The alphabetical does mimic the CloudFormation docs.

Reply:
Then autogenerate it is!  I'm assuming you'll put your snippet in the Travis yaml?